_case-studies/civil-rights-map-case-study.md

```diff
@@ -20,14 +20,15 @@ project_process: |
 project_results: |
   The Durham Civil and Human Rights History Map now has a beautifully-designed, and responsive, online home, which visitors and native Durhamites alike can use to tour the city. Savas Labs has also written a [number](/2015/06/10/d8-theming-basics.html) [of](/2015/07/06/map-in-drupal-8.html) [highly-viewed](/2015/05/18/mapping-geojson.html) [blog](/2015/09/03/sassy-drupal-theming-part-3.html) [posts](/2015/11/05/drupal-web-mapping.html) documenting what we've learned about Drupal 8 theming and mapping from the project.
 services_provided: |
-  - [Web map design](/blog/tag/cartography)
-  - [Drupal theming and site-building](/blog/tag/theming/)
-  - [Drupal contrib module development](/blog/tag/drupal/)
+  - [Web design](/services#design)
+  - [Drupal theming and site building](/blog/tag/theming/)
```

Comment:
I think it makes more sense to link theming to the tag for theming rather than link to the development service again - @chrisarusso thoughts?